README.md

```diff
@@ -50,14 +53,16 @@ You have to set up a step like this in your workflow (this assumes you've alread
     static-checking: localIsNew
 ```
 
+Now, when someone changes the version in `package.json` to `1.2.3` and pushes a commit with the message `<WHATEVER> 1.2.3` (eg. `Release 1.2.3` or `Bump version to v1.2.3`), output values are set (see Outputs below). Otherwise, no output is provided.
```

Comment:
> Otherwise, no output is provided.

Relates to #30.

Reply:
I'll solve issue #30 before merging this PR